main.py

```diff
@@ -16,4 +17,43 @@
     print(item)
 conn.close()
 diff = datetime.datetime.now() - start
-print('End {} seconds '.format(diff.total_seconds()))
+print('End {} seconds '.format(diff.total_seconds()))'''
+
+#Edited Code:
+class GetandSortData:
```

Comment:
Change to a better class name. Example; https://pspdfkit.com/blog/2018/naming-classes-why-it-matters-how-to-do-it-well/ and https://gist.github.com/hurricane-voronin/9ceccace0fd530bbf17c83b059c86eb7